src/Core/Common/FindMaxAndMinInImage/CMakeLists.txt

```diff
@@ -5,9 +5,11 @@ project(FindMaxAndMinInImage)
 find_package(ITK REQUIRED)
 include(${ITK_USE_FILE})
 
+find_package( VTK REQUIRED )
+include_directories(${VTK_INCLUDE_DIRS})
 
 add_executable(FindMaxAndMinInImage Code.cxx)
-target_link_libraries(FindMaxAndMinInImage ${ITK_LIBRARIES})
+target_link_libraries(FindMaxAndMinInImage ${ITK_LIBRARIES} ${VTK_LIBRARIES})
```

Comment:
I think you're missing the [`vtk_module_autoinit`](https://vtk.org/doc/nightly/html/group__module.html#gaa19497c3d949b3a93eaa13ed6df5d416) calls for this.